docker/Dockerfile

```diff
@@ -36,6 +36,6 @@ RUN find ./src -type f -exec touch {} \; \
 COPY ./docker/config.toml ./config.toml
 COPY ./docker/keyfiles /keyfiles
 
-COPY ./lint.sh ./test.sh ./coverage.sh ./
+COPY ./lint.sh ./test.sh ./coverage.sh ./audit.sh ./
```

Comment:
when source files are in the current working directory, you do not need to prefix with "./"
For the destination, if you want the destination to be in the current working directory, you can just do ".".

Reply:
https://docs.docker.com/engine/reference/builder/#copy